packages/suite/src/hooks/suite/useFirmware.ts

```diff
@@ -48,13 +52,39 @@ export const useFirmware = () => {
         modal.context === MODAL.CONTEXT_DEVICE &&
         modal.windowType === 'ButtonRequest_FirmwareCheck';
 
-    const isCurrentlyBitcoinOnly = hasBitcoinOnlyFirmware(originalDevice);
+    // Device may be wiped during firmware type switch because Universal and Bitcoin-only firmware have different vendor headers,
+    // except T1B1 and T2T1. There may be some false negatives here during custom installation.
+    // TODO: Determine this in Connect.
+    const willDeviceBeWiped = () => {
```

Comment:
is there a reason to have this in function?
It adds no semantic value when we have the variable, IMHO this is clear enough:
```
const deviceWillBeWiped = (
            shouldSwitchFirmwareType &&
            deviceModelInternal !== undefined &&
            ![DeviceModelInternal.T1B1, DeviceModelInternal.T2T1].includes(deviceModelInternal)
        );
```

Reply:
7782d9366e8e0553c7c56ed37314d20fa7409996